### PR TITLE
[CI] add CI for Windows platform

### PR DESCRIPTION
**Pull request by hrmthw**
(Thanks for sending a pull request! Please make sure to read the contribution guidelines, then fill out the blanks below.)

What changes were proposed in this pull request?
------------------------------------------------

Does this close any currently open issues?
------------------------------------------

How was this patch tested?
--------------------------

### CODE REVIEW

File: bentoml/cli/__init__.py [hunk 1]
```diff
@@ -353,7 +353,9 @@ def serve_gunicorn(
         if workers is None:
             workers = get_gunicorn_num_of_workers()
 
+        # Gunicorn only supports POSIX platforms
```
Comment (parano) [inline bentoml/cli/__init__.py:356]: maybe add an os platform check here? something like:

```
if os == "windows":
   _echo("The `bentoml server-gunicon` command is not supported on Windows. On windows platform, use `bentoml serve` for local API testing and docker for running production API endpoint: https://docs.docker.com/docker-for-windows/ ")
```

File: tests/cli/test_cli.py [hunk 2]
```diff
@@ -42,6 +44,7 @@ def test_run_command_with_input_file(bento_bundle_path):
     assert result.output.strip() == '3'
 
 
+@pytest.mark.skipif('not psutil.POSIX')
```
Comment (parano) [inline tests/cli/test_cli.py:47]: 👍

**Comment by hrmthw** [comment]
Temporarily disabled test_pip_install_saved_bundle for Windows to make CI happy.

**Comment by parano** [comment]
Great job fixing all the test issues on windows & travis ci! @hrmthw

**Pull request merged**